src/elements/content-sidebar/versions/VersionsItemButton.scss

```diff
@@ -42,5 +42,9 @@
     &.bcs-is-selected {
         background-color: $hover-blue-background;
         border-color: $bdl-box-blue;
+
+        .bcs-VersionsItem-info {
+            color: $bdl-gray;
```

Comment:
Does `color: inherit` work here to avoid duplicating the style declared on line 7 above?

Reply:
Yes it does, will send an updated commit in a bit